### ISSUE TITLE
Iceberg v1.2.0 breaks projects

### ISSUE THREAD

**Issue by jecisc** [pharo-vcs/iceberg#950]
I tried to use Iceberg v1.2.0 with MDL.

I loaded the project in my image via a Metacello script and Iceberg generated a .project and a .properties file. I committed them and now I cannot reload MDL because I get an error saying that it does not find the baseline.

MDL is a FileTree metadataless repository and I have the impression that since the commit of the files, Iceberg thinks it is a tonel repository.

Commit with the files: https://github.com/DuneSt/MaterialDesignLite/commit/dfe941d3b0e16cfad7fd45e6b17618faf623eac3

I'll revert those changes for now...

**Comment by jecisc** [comment]
I have the impression that:
- If I add the repository to Iceberg, I can use a repair action and add the format
- If I load a project from Metacello, the .project file is generated without the format and Iceberg will consider it as a Tonel project.

**Comment by guillep** [comment]
I'm checking it now. It is strange because we have a large battery of tests for the metacello integration and we covered such cases

**Comment by guillep** [comment]
Is this the expression you used

```
   Metacello new
    	githubUser: 'DuneSt' project: 'MaterialDesignLite' commitish: 'master' path: 'src';
    	baseline: 'MaterialDesignLite';
    	onWarningLog;
    	load
```

**Comment by jecisc** [comment]
I used development, else, yes.

**Comment by guillep** [comment]
It's strange, because when committing it's proposing me a `.properties` with

```
{
	#format : #filetree
}
```

**Comment by guillep** [comment]
Also, the fact that the project is loaded, it shows that even using the Metacello expression the project is correctly recognized as a filetree project.

**Comment by jecisc** [comment]
This is weird... I retried and now it exports the right datas... I did it with the same image and same snippet.

What is really weird is that before it failed for all my projects: https://github.com/DuneSt/MaterialColors/commit/21219e1efdb88e9caec7927be11ede7e3e3c3be2

Now it works for all of them.

Since I don't know how to reproduce I'll close it.

**Comment by jecisc** [comment]
I think I found where is the problem in what I have done.

Here is a gif showing a problem:

![2018-08-06_14-12-18](https://user-images.githubusercontent.com/9519971/43716156-55aad69a-9983-11e8-81d0-1e5f85fd3d1c.gif)

I loaded the project MaterialColors via Metacello and a tag. It generated the right files. Then I wanted to commit that to the development branch. Since there is not "merge with branch" repair option, I created a new branch from the tag then I checkout the development branch. After the checkout, the file .project is fine but the .properties is empty.
